ci/win_test_installer.cmd

```diff
@@ -12,6 +12,10 @@ echo %PATH%
 echo %PYTHONPATH%
 echo %NEURONHOME%
 
+:: Mitigation strategy -> for reasons uknown(thank you Windows), association.hoc.out may not be generated from previous step.
+:: If so, try again to generate it. No wait required like previous strategies, we rely on testing entropy from this point on.
+if not exist association.hoc.out (start /wait /REALTIME %cd%\ci\association.hoc)
```

Comment:
hmm...I must say I have forgotten conceptual control of this test i.e. how `association.hoc.out` is generated at this stage :)

Reply:
hehe, long story. First it was launched and then a wait (`ping 127.0.0.1 -n 10` then 15, 20 ..). 
Recently split the generation of the out file in another step and that made the error occur less. 
With this we launch it again if it hasn't been generated and test at the end (effectively waiting for all the other tests ~3 minutes, so no `ping` needed)